proof_of_play.go

```diff
@@ -238,36 +224,19 @@ func (p *proofOfPlay) confirm(popReq *PoPRequest) error {
 		}
 	}
 
-	return p.makePoPRequest(req, popReq, adId, "pop")
+	return p.makePoPRequest(req, popReq, popReq.AdId, popReq.Type)
```

Comment:
If these are all part of popReq now, do we need to pass them in? can we just get them from popReq in makePoPRequest?